### PR TITLE
fixes docker compose example with correct nom image & instructions

### PR DESCRIPTION
**Pull request by ravikiran-sulikeri**

### CODE REVIEW

File: modules/ROOT/pages/docker/compose.adoc [hunk 1]
```diff
@@ -95,15 +97,15 @@ services:
 
   server:
     hostname: server
-    image: neo4j-ops-manager:latest
+    image: neo4j/neo4j-ops-manager-server:latest
```
Comment (eve-bright) [inline modules/ROOT/pages/docker/compose.adoc:100]: Image also needs updated in previous section for the server compose template

**Review by eve-bright — CHANGES_REQUESTED** [review]

**Review by eve-bright — APPROVED** [review]

File: modules/ROOT/pages/docker/compose.adoc [hunk 2]
```diff
@@ -69,6 +69,8 @@ docker compose -f <compose file> up <folder for context, typically current folde
 ====
 Following is an example Docker compose file that can be used to start up a NOM server and Enerprise Neo4j Persistence system without any customization. You can use it to quickly setup a NOM test environment and add DBMSs for monitoring:
 
+IMPORTANT: Create a directory `~/.nom/ssc` before running Docker compose. Certificates will be generated as by products of running below compose file at `~/.nom/ssc` directory which can be used to configure the NOM agent as given xref:addition/index.adoc#configure[here].
```
Comment (AlexicaWright) [inline modules/ROOT/pages/docker/compose.adoc:72]:
```suggestion
IMPORTANT: Create a directory `~/.nom/ssc` before running Docker compose. 
Certificates are generated as by-products of running below compose file at `~/.nom/ssc` directory which can be used to configure the NOM agent as given xref:addition/index.adoc#configure[here].
```

File: modules/ROOT/pages/docker/introduction.adoc [hunk 3]
```diff
@@ -1,13 +1,13 @@
 = Introduction
 :description: describes the containerization of the NOM server
 
-For every NOM release a Docker image for the NOM server is published to the https://hub.docker.com/r/neo4j/neo4j-ops-manager/tags[Docker Hub verified repository].
+For every NOM release a Docker image for the NOM server is published to the https://hub.docker.com/r/neo4j/neo4j-ops-manager-server/tags[Docker Hub verified repository].
```
Comment (AlexicaWright) [inline modules/ROOT/pages/docker/introduction.adoc:4]:
```suggestion
For every NOM release, a Docker image for the NOM server is published to the https://hub.docker.com/r/neo4j/neo4j-ops-manager-server/tags[Docker Hub verified repository].
```

File: modules/ROOT/pages/docker/server.adoc [hunk 4]
```diff
@@ -11,14 +11,14 @@ Refer to *xref:../installation/server.adoc[Installation]* for the list of config
 
 == Security
 NOM server security involves configuring keys and certificates for the TLS connections that the server makes with the UI and the agents. 
-For NOM server container keys and certificates should be mounted into the container as shown below:
+For NOM server container, keys and certificates should be mounted into the container as shown below:
```
Comment (AlexicaWright) [inline modules/ROOT/pages/docker/server.adoc:14]:
```suggestion
For NOM server container, keys, and certificates should be mounted into the container as shown below:
```

**Review by AlexicaWright — COMMENTED** [review]
Just some editorial comments.

**Review by AlexicaWright — COMMENTED** [review]
Just one last case of present tense.

**Review by AlexicaWright — APPROVED** [review]
👍 LGTM, thank you @ravikiran-sulikeri !

**Pull request merged**